### PR TITLE
tweak algorithm and tool tip

### PR DESCRIPTION
**Pull request by MaxBittker**
fixes 1 length matches like eric's false positive, 
adds repo link to tool tip, 
prevents empty owners from rendering

### CODE REVIEW

**Comment by MaxBittker** [comment]
mar-31-2017 13-13-48

File: src/sentry/static/sentry/app/components/group/suggestedOwners.jsx [hunk 1]
```diff
@@ -6,7 +6,7 @@ import Avatar from '../avatar';
 import TooltipMixin from '../../mixins/tooltip';
 import ApiMixin from '../../mixins/apiMixin';
 import GroupState from '../../mixins/groupState';
-
+// import {CommitLink} from '../../views/releases/releaseCommits';
```
Comment (MaxBittker) [inline src/sentry/static/sentry/app/components/group/suggestedOwners.jsx]: im sorry for this - i wanted to save the relative path

File: src/sentry/static/sentry/app/views/releases/releaseCommits.jsx [hunk 2]
```diff
@@ -9,13 +9,9 @@ import ApiMixin from '../../mixins/apiMixin';
 
 import {t} from '../../locale';
 
-const ReleaseCommit = React.createClass({
+const CommitLink = React.createClass({
```
Comment (MaxBittker) [inline src/sentry/static/sentry/app/views/releases/releaseCommits.jsx]: could probably break this out into its own component

File: src/sentry/static/sentry/app/components/group/suggestedOwners.jsx [hunk 3]
```diff
@@ -70,30 +70,41 @@ const SuggestedOwners = React.createClass({
...
   },
 
+  assignTo(member) {
+    this.api.assignTo({id: this.props.event.groupID, member: member});
+  },
+
   renderCommitter({author, commits}) {
     return (
-      <span key={author.id} className="avatar-grid-item tip" title={
+      <span key={author.id} className="avatar-grid-item tip" onClick={()=>this.assignTo(author)} title={
```
Comment (macqueen) [inline src/sentry/static/sentry/app/components/group/suggestedOwners.jsx]: i usually put spaces like `() => this.assignTo..` but looks like we already have a few cases of spacing like this so whatevs

File: src/sentry/static/sentry/app/components/group/suggestedOwners.jsx (hunk 1 above)
Comment (macqueen) [inline src/sentry/static/sentry/app/components/group/suggestedOwners.jsx]: i'm not totally sure what you mean. why is this necessary?

File: src/sentry/static/sentry/app/components/group/suggestedOwners.jsx [hunk 4]
```diff
@@ -70,30 +70,41 @@ const SuggestedOwners = React.createClass({
...
+            <strong className="time-label">
+              {`${author.name}: `}
+            </strong>
+            <div className="commit-list">
+              {commits.map( c => {
+                  return (
+                 <span key={c.id}>
+                  <span>{c.message} : </span>
+                  <span>{moment(c.dateCreated).fromNow()}</span>
+                  {/* <CommitLink commitId={c.id} repository={c.repository}/> */}
```
Comment (macqueen) [inline src/sentry/static/sentry/app/components/group/suggestedOwners.jsx]: not sure if you meant to leave this commented out? or just delete it?

File: src/sentry/static/sentry/app/views/releases/releaseCommits.jsx (hunk 2 above)
Comment (MaxBittker) [inline src/sentry/static/sentry/app/views/releases/releaseCommits.jsx]: i mean file

**Review by benvinegar — APPROVED** [review]
Gross.

**Review by macqueen — APPROVED** [review]
looks fine to me other than commented out stuff that should (i think?) be removed
